components/transaction/internal/app/command/create-assetrate_test.go

```diff
@@ -0,0 +1,61 @@
+package command
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	a "github.com/LerianStudio/midaz/components/transaction/internal/domain/assetrate"
```

Comment:
please, use **ar** for **asset_rate** because we are using **a** for **accounts**, just to normaliza the imports.